### PR TITLE
API Performance degrade - Closes #2817

### LINKED ISSUE #2817: API Performance degrade
### Expected behavior
API Performance should be tolerant`(100-600ms)`.
### Actual behavior
Following API Performance have been degraded
`GET dapps`
`POST signatures`
`GET voters`
`GET accounts`
### Steps to reproduce
https://rpm.newrelic.com/accounts/2081667/applications/130457745/transactions?tw%5Bend%5D=1549281180&tw%5Bstart%5D=1549259580
### Which version(s) does this affect? (Environment, OS, etc...)
1.5.0

### PR DESCRIPTION
**Pull request by lsilvs**
### What was the problem?
API was using `extended` version of accounts

### How did I fix it?
I moved `productivity` to the non-extended version and removed the `extended` from API calls

### How to test it?
Run test suite

### Review checklist

* The PR resolves #2817 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: test/unit/storage/entities/account.js [hunk 1]
```diff
@@ -180,6 +180,14 @@ describe('Account', () => {
...
 			'rewards',
 			'vote',
+			'u_username',
+			'u_isDelegate',
+			'u_secondSignature',
+			'u_nameExist',
+			'u_multiMin',
+			'u_multiLifetime',
+			'u_balance',
+			'productivity',
```
Comment (MaciejBaj) [inline test/unit/storage/entities/account.js]: returning productivity for every account?

File: storage/sql/accounts/get.sql [hunk 2]
```diff
@@ -28,7 +28,20 @@ SELECT
...
+	"u_secondSignature"::int::boolean,
+	"u_nameexist"::int::boolean as "u_nameExist",
+	"u_multimin" as "u_multiMin",
+	"u_multilifetime" as "u_multiLifetime",
+	"u_balance",
+	case
+	when
+		"producedBlocks" + "missedBlocks" = 0 then 0
+	else
+		(("producedBlocks"::float / ("producedBlocks" + "missedBlocks")) * 100.0)::integer
```
Comment (MaciejBaj) [inline storage/sql/accounts/get.sql]: Then the logic of logic/accounts.js, `calculateProductivity` function is redundant.

File: storage/sql/accounts/get.sql [hunk 3]
```diff
@@ -28,7 +28,20 @@ SELECT
...
+	"u_username",
+	"u_isDelegate"::int::boolean,
+	"u_secondSignature"::int::boolean,
+	"u_nameexist"::int::boolean as "u_nameExist",
+	"u_multimin" as "u_multiMin",
+	"u_multilifetime" as "u_multiLifetime",
+	"u_balance",
+	case
+	when
+		"producedBlocks" + "missedBlocks" = 0 then 0
```
Comment (MaciejBaj) [inline storage/sql/accounts/get.sql]:
```suggestion
		"isDelegate" = 0 then 0
```

File: test/unit/storage/entities/account.js (hunk 1 above)
Comment (lsilvs) [reply test/unit/storage/entities/account.js]: It's used for delegate accounts. It was already present in `extended` accounts but as the calculation is not costly I decide to move to `basic` accounts.

File: storage/sql/accounts/get.sql (hunk 2 above)
Comment (lsilvs) [reply storage/sql/accounts/get.sql]: `calculateProductivity` was removed as part of 1.5 development

File: storage/sql/accounts/get.sql (hunk 3 above)
Comment (lsilvs) [reply storage/sql/accounts/get.sql]: If the a delegate account has `producedBlocks` and `missedBlocks` equal to zero, the `else` statement will fail due to division by zero.

Comment (nazarhussain) [reply storage/sql/accounts/get.sql]: I tend to believe on our application logic that produced blocks and missed blocks will only be incremented for delegate accounts.

Comment (lsilvs) [reply storage/sql/accounts/get.sql]: But you can still have a delegate account with `producedBlocks` and `missedBlocks` equal to zero

**Pull request merged**